query-engine/schema-builder/src/input_types/fields/data_input_mapper/update.rs

```diff
@@ -26,7 +26,7 @@ impl DataInputFieldMapper for UpdateDataInputFieldMapper {
             TypeIdentifier::String => InputType::object(update_operations_object_type(ctx, "String", sf, false)),
             TypeIdentifier::Boolean => InputType::object(update_operations_object_type(ctx, "Bool", sf, false)),
             TypeIdentifier::Enum(e) => {
-                InputType::object(update_operations_object_type(ctx, &format!("Enum{e}"), sf, false))
+                InputType::object(update_operations_object_type(ctx, &format!("Enum{e:?}"), sf, false))
```

Comment:
This changes the name of the schema type and it's _breaking_. It used to be the enum name and is now a u32. I suppose you could find the enum name and pass it as before using the `InternalDataModel` stored in the `BuilderContext`.